R/aadoc.R

```diff
@@ -238,3 +238,16 @@ NULL
 #' @name temperature
 #' @family capability identifiers
 NULL
+
```

Comment:
Please remove the `Identifiers for quantities in the core` chunk of code.

Reply:
👍